apps/fabric-website/src/pages/HomePage/HomePage.base.tsx

```diff
@@ -210,7 +283,7 @@ export class HomePageBase extends React.Component<IHomePageProps, IHomePageState
           <figure className={this._classNames.oneHalf}>
             <ul className={this._classNames.usageIconList}>{this._renderUsageIconList()}</ul>
             <figcaption>
-              <strong>+ many additional Microsoft sites and products</strong>
+              <strong>+ 52 additional Microsoft sites and products</strong>
```

Comment:
Thanks for making this change, but let's please keep this as "many". Specific numbers like this are brittle--it could grow to 53+ as a result of Build 😂

I actually got specific guidance to use "many" from marketing for the Build talk--where I'm using a similar graphic--because specific numbers can actually make this sound less credible ("the Fabric team doth protest too much").

Reply:
The specific number was actually a request from Angela, but if you were told not to use it for your //build talk, I'll hold off on adding it here too.